app/backend/dru/views.py

```
@@ -14,25 +13,19 @@
     DRUTypeSerializer,
     RegionSerializer,
 )
+from .models import (
+    DRU,
+    DRUType,
+)
 from dru.models import DRUType, DRU
```

Ruff (0.11.9)

Redefinition of unused `DRUType` from line 18

(F811)